### PR TITLE
Refactored app.js, resolved linter errors and upgraded dependencies

### PR DESCRIPTION
**Pull request by specious**

### CODE REVIEW

**Comment by huttarl** [comment]
@specious How do you create a new PR without your previous one being closed? I can't seem to push any commits without them being included in the PR I created earlier.

**Comment by specious** [comment]
The key is:

specious wants to merge 9 commits into codeforamerica:master from specious:refactor

I can create a new branch, again from `master`:
```
git checkout master
git checkout -b feature3
git commit
[...]
```
Then I can open a request to merge `specious:feature3` into `codeforamerica:master`.

What if I already submitted a pull request from `master`?  Then I'd be careful to make sure I'm creating a new feature branch from `upstream/master` and not my `master`.

Here's a great [answer](http://stackoverflow.com/a/32171246/12899010) on Stack Overflow.

**Comment by huttarl** [comment]
@specious 👍 Thanks, that's very helpful.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/codeforamerica/click_that_hood.png?branch=master)](click_that_hood)
+[![Build Status][https://img.shields.io/travis/codeforamerica/click_that_hood/master.svg]][https://travis-ci.org/codeforamerica/click_that_hood]
```
Comment (k-nut) [inline README.md]: What is the advantage of shields.io over the travis hosted badge?

Comment (specious) [inline README.md]: Thanks for asking.  Shields [provides](
https://github.com/badges/shields#solving-the-problem) consistent, resolution-independent informational images.

**Comment by specious** [comment]
@mwichary, could you please review this?

**Comment by ethomas12** [comment]
Thanks @specious,, we worked on Click that Hood in June and I have been waiting to see someone else's work that passed inspection because I could not figure out what went wrong.

**Comment by specious** [comment]
Pull requests are piling up for this project.  Perhaps @mwichary or @ycombinator could review them?

**Comment by specious** [comment]
Ping?

**Comment by specious** [comment]
I hate to nag anybody, but if someone could make me the maintainer of this project I would love to merge all the pull requests.

**Comment by fnogatz** [comment]
Some weeks ago, I sent an email to @mwichary to ask about the current state and future plans for *Click that 'hood*. He is not involved in this project anymore and asked to directly contact *Code for America* instead and ask for these things and the missing LICENSE. Unfortunately, they still have not answered my email, so I just [tried again via Twitter](https://twitter.com/ulmerleben/status/1418895395086798850). I will let you knpw when I get a reply...

**Comment by specious** [comment]
I would like to know who the best person to contact is.

**Comment by fnogatz** [comment]
Me too. As of now, no one has answered :(

**Comment by fnogatz** [comment]
Hey @specious, in the meantime, I got owner permissions and transferred the repository in the namespace of *Code for Germany*. I already labelled all issues and open PRs and suggest to first merge all things tagged as  ([list here](https://github.com/codeforgermany/click_that_hood/pulls?q=is%3Aopen+is%3Apr+label%3Ainfrastructure)) – beginning with this one? :) All the things related to refactoring in general, updating dependencies, etc. should be done only after the three open PRs have been processed.

@specious, I added you to the project, so you can revise this PR again and either merge it or request a review!

**Comment by specious** [comment]
@fnogatz, I just looked over your recent work on the master branch. Thank you for organizing this effort.

Just give me a minute to take a look at all the branching, so I can merge this correctly.

**Pull request merged**